test/models/patient_notification_eligibility_test.rb

```diff
@@ -0,0 +1,650 @@
+    create(:assessment, patient: patient, symptomatic: false, created_at: default_days_ago(2))
+    assert_eligible(patient)
+  end
+
+  test 'exposure send report with continuous exposure' do
+    # patient was created more than 24 hours ago
+    patient = create(
+      :patient,
+      preferred_contact_method: 'E-mailed Web Link',
+      monitoring: true,
+      purged: false,
+      isolation: false,
+      created_at: default_days_ago(2),
+      continuous_exposure: true
+    )
+    # patient has asymptomatic assessment more than 1 day ago but less than 7 days ago
+    create(:assessment, patient: patient, symptomatic: false, created_at: default_days_ago(2))
+    assert_eligible(patient)
+  end
+
+  test 'Patients in differing timezones are eligible when expected' do
+    [
+      { monitored_address_state: 'Florida' },
+      { monitored_address_state: 'Colorado' },
+      { monitored_address_state: nil, address_state: 'California' },
+      {}
+    ].each do |patient_params|
+      patient = create(
+        :patient,
+        {
+          preferred_contact_method: 'E-mailed Web Link',
+          continuous_exposure: true
+        }.merge(patient_params)
+      )
+      Timecop.freeze(Time.now.getlocal(patient.address_timezone_offset).noon) do
+        assert_eligible(patient)
+        # assessment right before the start of the valid reporting period
+        create(
+          :assessment,
+          patient: patient,
+          # converting to UTC because these DB times are assumed to be saved in UTC
+          created_at: Time.now.getlocal(patient.address_timezone_offset).yesterday.end_of_day.utc
+        )
+        assert_eligible(patient)
+        # assessment right after the start of the valid reporting period
+        create(
+          :assessment,
+          patient: patient,
+          created_at: Time.now.getlocal(patient.address_timezone_offset).beginning_of_day.utc
+        )
+        assert_ineligible(patient)
+      end
+    end
+  end
+
+  test 'configuring reporting period affects eligibility' do
+    # patient was created more than 24 hours ago
+    patient = create(
+      :patient,
+      preferred_contact_method: 'E-mailed Web Link',
+      monitoring: true,
+      purged: false,
+      isolation: false,
+      created_at: default_days_ago(4),
+      continuous_exposure: true
+    )
+    assert_eligible(patient)
+    ADMIN_OPTIONS['reporting_period_minutes'] = 1440 * 7 # 1 week
+
+    create(
+      :assessment,
+      patient: patient,
+      symptomatic: false,
+      created_at: (Time.now.getlocal(patient.address_timezone_offset) - 7.days).end_of_day
+    )
+    assert_eligible(patient)
+    create(
+      :assessment,
+      patient: patient,
+      symptomatic: false,
+      created_at: (Time.now.getlocal(patient.address_timezone_offset) - 6.days).beginning_of_day
+    )
+    assert_ineligible(patient)
+  end
+
+  # test 'morning preferred contact time' do
```

Comment:
Why are all of these commented out as well?

Reply:
Removing these commented out tests because these cases should now be tested with tests for the new `within_preferred_contact_time` scope located in `patient_test.rb`